Gemfile

```diff
@@ -41,7 +41,6 @@ gem "manageiq-api-client",            "~>0.1.0",       :require => false
 gem "manageiq-network_discovery",     "~>0.1.2",       :require => false
 gem "mime-types",                     "~>2.6.1",       :path => "mime-types-redirector"
 gem "more_core_extensions",           "~>3.3"
-gem "nakayoshi_fork",                 "~>0.0.3"  # provides a more CoW friendly fork (GC a few times before fork)
```

Comment:
Is this still useful for other uses of fork, such as the parallel gem?

Reply:
I doubt it.  The only place that cared about trying to eek out more shared memory was worker creation.  We can keep it but this line was not added for users of parallel .